46.md

```diff
@@ -67,7 +67,7 @@ The signer key will always be the key of the user who controls the signer device
 ### Methods
 
 - **connect**
-  - params: [`pubkey`, `secret`]
+  - params: [`pubkey-hex`, `secret`]
```

Comment:
given that connect can be from either the signer or the client, do we know whether `pubkey-hex` changes in each scenario?

also any idea why we need `secrect`? assuming this is the client's private key..?

Reply:
no, `secret` is a secret that can be provided out of band to the client from the signer; this is an easy way to identify the client in front of the signer.

Scenario:

A company provides authorization to 100 employees to sign using the conpany's key
They create 100 secrets and give a secret to each employee.
Each secret is redeemed without prior knowledge of which client-pubkey will do the redemption.

Reply:
where is this detailed?

is the secret for `connect` that is from client to signer only? or is it also for `connect` that is from signer to client as well?

I don't think i'm understanding, from the example you shared it seems like it is the employee's private key that can sign on behalf of the company's key. which sounds like the private key on the client side.